modules/caddyhttp/templates/tplcontext.go

```diff
@@ -406,7 +406,7 @@ func (c TemplateContext) funcHTTPError(statusCode int) (bool, error) {
 // The supported time formats: https://pkg.go.dev/time#pkg-constants
 func (c TemplateContext) formatHuman(function, data string) (string, error) {
 
-	switch function {
+	switch function[:4] {
```

Comment:
I'd say it's probably better to `strings.Split(function, ":")` instead. Relying on the length of the string seems awkward.

Reply:
I meant - split and store the parts as a variable _before_ the switch, then switch on the first part, and use the second part later if it's there. You're still doing a string slice later inside of the `time` branch of the switch right now

Reply:
Like:

```
parts := strings.Split(function, ":")
switch parts[0] {
    ...
case "time":
    ...
    if len(parts) > 1 {
        timelayout = parts[1]
    }
```